### ISSUE TITLE
AppImage crashes on startup on Debian Bookworm

### ISSUE THREAD

**Issue by khimaros** [BespokeSynth/BespokeSynth#783]
## Describe the bug

app crashes immediately on startup on debian bookworm:

```
$ ./bespokesynth-1.1.0-lp153.3.2.Build3.1.glibc2.29-x86_64.AppImage 2>&1 | head
terminate called after throwing an instance of 'std::runtime_error'
  what():  Application directory not found. Please reinstall Bespoke.
terminate called recursively
[...]
terminate called recursively
Segmentation fault
```

## Build info

not possible, crashes immediately.

## Steps To Reproduce

download the latest 1.1.0 AppImage, `chmod +x` and then execute.

## Expected vs. actual behavior

the `-current` AppImage `1.1.0+git.62.9a6d95a7-lp153.2.1.Build3.1.glibc2.29-x86_64` also crashes on startup.

the nightly tarball from `2022-03-13` launches fine but has a broken scripting environment https://github.com/BespokeSynth/BespokeSynth/issues/778

previous version (1.1.0 beta?) AppImage `1.1.0-lp152.1.1.Build1.1.glibc2.25-x86_64` launches and works fine

**Comment by KottV** [comment]
Ouch. Thanks for pointing. I updated AppImage deps for openSUSE 15.3 but forgot to patch bespoke for custom `datadir`
Fixed package should arrive in half-hour or so.

BTW, why do you use AppImage instead of native .deb?

**Comment by khimaros** [comment]
@KottV thank you!

it's partly a security / harm reduction practice.

tl;dr, installing a .deb means you are running arbitrary commands as root, outside of a sandbox.

my order is something like:

1. official debian repo
2. flatpak (typically from flathub)
3. appimage
4. upstream binary (as unprivileged user)
5. built from source (as unprivileged user)
6. unofficial debian repo

i typically only use [6] as a last resort because there is less adherence to debian policy: some packages do unsavory things like modify your sources.list or add new signing keys.

**Comment by KottV** [comment]
you may see and inspect the sources of package

**Comment by khimaros** [comment]
@KottV that's true! but security practices are more likely to be followed when also optimized for laziness :)

**Comment by khimaros** [comment]
@KottV -- i tested build `1.1.0-lp153.3.2.Build4.1.glibc2.29-x86_64` but it is crashing in the same way.

**Comment by KottV** [comment]
@khimaros please try bespokesynth-1.1.0-lp153.4.1.Build5.1.glibc2.29-x86_64.AppImage

**Comment by khimaros** [comment]
that fixed it and scripting seems to be working in this version too. thank you!

**Comment by KottV** [comment]
bespoke-current appimage with fixed script load should arrive soon
